linter/linter.go

```diff
@@ -1345,6 +1345,13 @@ func (l *Linter) lintInfixExpression(exp *ast.InfixExpression, ctx *context.Cont
 
 	switch exp.Operator {
 	case "==", "!=":
```

Comment:
can we add a test for that? or a fiddle is this for something like `req.backend == backend_F`?